terminal.go

```diff
@@ -1437,6 +1450,20 @@ func (g *LightningTerminal) showStartupInfo() error {
 	return nil
 }
 
+// connectLND sets up LiT's LND connection.
+func (g *LightningTerminal) connectLND(bufListener *bufconn.Listener) error {
```

Comment:
Could turn this into a function by taking the `cfg` as a parameter and returning the conn. Then we could place this function and the `dialBackend()` and `dialBufConnBackend()` into a different file to reduce the size of this one a bit. Maybe `lnd_connection.go` or something like that?